simplifier/world/nodes.py

```diff
@@ -114,9 +114,11 @@ def simplify(self):
         for node in list(self.world.iter_postorder(self)):
             if isinstance(node, BitwiseNegate):
                 node.dissolve_negation()
+        from simplifier.range_simplifier import RangeSimplifier
```

Comment:
What I meant was to move this import into line 113 so that the imports are at least at the beginning of the function. Sorry for the confusion :)

Reply:
Oh! I've moved the import to the beginning of the function.